pkg/result/processors/invalid_issue_test.go

```diff
@@ -31,6 +31,21 @@ func TestInvalidIssue_Process(t *testing.T) {
 				{FromLinter: "typecheck"},
 			},
 		},
+		{
+			desc: "typecheck and Go file",
```

Comment:
Might be worth clarifying this
```suggestion
			desc: "keep only typecheck issues if any exist",
```